styles/designs/carnival/parts/_table-components.scss

```diff
@@ -253,6 +253,14 @@ $Table__BottomBorder--AfterTable: (
     )
 );
 
+$Table__Row--AlternateShading: ( //nursing external (even rows)
+    _name: "LightGrayShading",
```

Comment:
Can you name it `AlternateShadingTableRow`? Other books can use different color, so the name shoudl be more general.